### ISSUE TITLE
SlowQueryEvent can support get sql with bind values? 

### ISSUE THREAD

**Issue by xu1009** [ebean-orm/ebean#2201]
i am using SlowQueryListener in ebean，but SlowQueryEvent  can only get sql without bind values，it is necessary to get sql with bind values, can it support?

**Comment by rbygrave** [comment]
The issue here is that we have the intention of semi automated query plan capture that can run in production and this would be much much better than the SlowQueryListener.

If we look at CQueryPlan: https://github.com/ebean-orm/ebean/blob/master/ebean-core/src/main/java/io/ebeaninternal/server/query/CQueryPlan.java#L350

... a lot of this is already in place in terms of capturing the actual bind values for slow queries and in the background collecting the query plan when desired.  So we have a lot of this in place already in the internals but it still isn't quite as easy to use as desired yet.

In other words, logging bind values for the SlowQueryListener is not really anywhere near as good as doing the proper solution (which I am still working on).

**Comment by rbygrave** [comment]
Related to https://github.com/ebean-orm/ebean/issues/1857

**Comment by rbygrave** [comment]
Hi @xu1009  I think we should close this issue and add to the comments / documentation at #1857 

I have put in there some details on how we can automatically collect query plan execution in 1857.

Cheers, Rob.

**Comment by rbygrave** [comment]
Closing as a duplicate of #1857
